src/pages/settings/email-settings/EmailSettings.tsx

```diff
@@ -374,7 +374,6 @@ export function EmailSettings() {
                 }
                 readonly={disableSettingsField('gmail_sending_user_id')}
                 errorMessage={errors?.errors['settings.gmail_sending_user_id']}
-                staleTime={1}
```

Comment:
@beganovich @turbo124 The Users Selector on the Email Settings page had a staleTime of 1, which basically means that the API will be triggered constantly on every page focus, etc. However, any change that needs to be applied to any User Selector will be done through invalidation on the Users Create/Edit pages. 

All other User selectors have staleTime by default Infinity so I removed for this one also.